code/game/objects/storage/coat.dm

```diff
@@ -48,3 +48,10 @@
 	else if(istype(over_object, /obj/structure/table)) //empty on table
 		return hold.MouseDropFrom(over_object)
 	return ..()
+
+/obj/item/clothing/suit/storage/AltClick()
```

Comment:
AltClick can take a `user` parameter, use that instead.

Reply:
Oh, it totally can? I was wondering that.